vault/cmd/m3-vault/main.go

```diff
@@ -181,17 +179,21 @@ func vaultInitAndUnseal() chan error {
 			return
 		}
 		log.Println("Vault enabled secrets kv successfully")
-
-		//<-storeConfInK8s(rootToken)
-
 	}()
 	return doneCh
 }
 
 func isVaultReadyRetry(address string) (*vapi.Client, error) {
 	currentTries := 0
+	totalRetries, _ := strconv.Atoi(env.Get("TOTAL_INIT_RETRIES", "5"))
```

Comment:
can you please add a case to handle error e.g.  `TOTAL_INIT_RETRIES -> "string"`
else the for will break badly if that happens, which could happen.